hack/os-packages/Makefile

```diff
@@ -1,17 +1,20 @@
 RPM_PACKAGES_GETTER_IMAGE = mesosphere/rpm-artifacts-getter
-RPM_FIPS_PACKAGES_GETTER_IMAGE = mesosphere/rpm-artifacts-getter-fips
-
+CENTOS_BASE_IMAGE=mesosphere/centos:7.9.2009.minimal
 KUBERNETES_RPM_PATCH = 0
 
 .PHONY: os-packages-artifacts
-os-packages-artifacts: download-all-rpms artifacts/os-packages_$(DEFAULT_KUBERNETES_VERSION_SEMVER)_x86_64_rpms.tar.gz
+os-packages-artifacts: download-all-rpms artifacts/os-packages_$(DEFAULT_KUBERNETES_VERSION_SEMVER)_x86_64_rpms.tar.gz artifacts/os-packages_$(DEFAULT_KUBERNETES_VERSION_SEMVER)_x86_64_fips_rpms.tar.gz
 
 artifacts/os-packages_$(DEFAULT_KUBERNETES_VERSION_SEMVER)_x86_64_rpms.tar.gz:
 	cd artifacts/os-packages_$(DEFAULT_KUBERNETES_VERSION_SEMVER)_el7_x86_64_rpms && tar -czf ../os-packages_$(DEFAULT_KUBERNETES_VERSION_SEMVER)_x86_64_rpms.tar.gz *
 
+artifacts/os-packages_$(DEFAULT_KUBERNETES_VERSION_SEMVER)_x86_64_fips_rpms.tar.gz:
+	cd artifacts/os-packages_$(DEFAULT_KUBERNETES_VERSION_SEMVER)_el7_x86_64_fips_rpms && tar -czf ../os-packages_$(DEFAULT_KUBERNETES_VERSION_SEMVER)_x86_64_fips_rpms.tar.gz *
+
 .PHONY: download-all-rpms
 download-all-rpms:
-	$(MAKE) rhel_major_version=7 fips=0 bastion=0  download-rpms
+	$(MAKE) rhel_major_version=7 fips=0 download-rpms
```

Comment:
removed `bastion=0` parameter as it was not used. It must have added for future purpose. Please let me know if I should keep it and how are we going to use it in future.

Reply:
Thanks for cleaning it up